_data/tasks.yml

```diff
@@ -171,6 +171,7 @@ toc:
   - docs/tasks/administer-cluster/ip-masq-agent.md
   - docs/tasks/administer-cluster/dns-custom-nameservers.md
   - docs/tasks/administer-cluster/pvc-protection.md
```

Comment:
`pvc-protection.md` file was renamed. Could it be deleted from `tasks.yml`?

Reply:
oh, i forgot it. done, thanks